packages/dashboards/app/styles/navi-dashboards/components/navi-collection.less

```
@@ -55,6 +55,10 @@
 
     .action {
       margin-right: 10px;
+
+      .modal {
+        visibility: visible !important;
```

Does this work? We didn't need `!important` in the directly styles:
```suggestion
        visibility: visible;
```

---

without it it got overriden by the row visibility

---

you might need to write a more specific selector

---

targeted modal.is-active and its working now